### PR TITLE
{math}[foss/2022a] graph-tool v2.55

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

replaces #17389

* [x] #17388 for `cairomm/1.16.2`

### CODE REVIEW

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/004ab03a5d57f58205b0616c2793bfe4 for a full test report.

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'cstd': 'c++17'}
+
+source_urls = ['https://downloads.skewed.de/%(name)s/']
+sources = [SOURCE_TAR_BZ2]
+checksums = ['75728e55b774af734954cd48f3d80ba64cf08864ef6143408f3a7422474834b4']
+
+builddependencies = [
+    ('gawk', '5.1.1'),
+    ('pkgconf', '1.8.0'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb:27]: Use pkg-config instead

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Author J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb:3]: This might have a better fit in configuremakepythonpackage

**Review by akesandgren — DISMISSED** [review]

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb:27]: Why? We switched to `pkgconf` for 2022a easyconfigs.

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb (hunk 2 above)
Comment (branfosj) [reply easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb:3]: `ConfigureMakePythonPackage` uses a python command at the configure stage (see https://github.com/easybuilders/easybuild-easyblocks/blob/develop/easybuild/easyblocks/generic/configuremakepythonpackage.py#L60) and that fails with `unknown option --enable-openmp`.

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb:27]: Sorry, got them mixed up (again)

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb (hunk 2 above)
Comment (akesandgren) [reply easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb:3]: Doh, forgot about that...

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @branfosj!

**Pull request merged**